### DISCUSSION TITLE
Daily 12/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#185] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer estuve hasta tarde estudiando pero toque poco el código, estuve profundizando en teoría. Hoy voy a avanzar con mis tareas asignadas y si termino asignarme un par mas.

**Comment by NataliaAlvarezIspc** [comment]
Ayer pude correr la aplicación sin problemas, luego de que me haya tirado un error el día anterior. Participé del encuentro de proyecto integrador y de la clase de inglés. Estuve haciendo tarea de inglés. También estuve viendo videos para la parte de registración y empecé a trabajar en la activity para que se conecte a la base de datos, me generó algunos errores así que lo deje para continuarlo hoy.
Hoy estuve viendo más videos y a la tarde nos juntamos con Braian para resolver lo que estaba mal. Así que ya está hecha una registración inicial que se carga en la base de datos. Voy a seguir trabajando en validaciones, responsive, etc de la registración. También voy a participar de la clase sincrónica de hoy.
